test/junit/scala/tools/nsc/interpreter/CompletionTest.scala

```diff
@@ -120,36 +120,6 @@ class CompletionTest {
     checkExact(completer, """class C { def +++(a: Any) = 0; def ---(a: Any) = 0; this.++""")("+++")
   }
 
-  @Test
-  def camelCompletions(): Unit = {
```

Comment:
Is there hope to ever bring this back...? Maybe as an upstream contribution?

Reply:
Ah, I saw now https://github.com/scala/scala-dev/issues/691#issuecomment-617550026